docs/csc-source-resolution.md

```diff
@@ -0,0 +1,14 @@
+# CatalogSourceConfig Source Resolution
+
+v2 of the `CatalogSourceConfig CRD` expects a `source` that matches the name of an `OperatorSource` on cluster that all packages must originate from. If a CatalogSourceConfig is created without specifying a `source`, the Marketplace Operator will attempt to update the `source` field with the name of a valid `OperatorSource` that contains the list of `packages`.
+
+This document will describe how the Marketplace Operator will attempt to reconcile invalid and missing `sources` and the phase that the `CatalogSourceConfig` will eventually reach. Any `CatalogSourceConfig` placed in the `Configuring` phase can eventually be resolved if the described errors are addressed.
+
+## Example Scenarios
+
+1. If a `CatalogSourceConfig` defines a `source` that does not exist on the cluster, the `CatalogSourceConfig` will be placed in the `Configuring` phase and alert the user that the `source` does not exist.
```

Comment:
"alert the user" seems to imply some sort of notification

Reply:
I will change `alert` to `update the `phase.message`.